go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/openebs/api
+module github.com/openebs/api/v2
 
 go 1.13
```

Comment:
go version need to be 1.14.7 or above, because of [CVE-2020-16845](https://nvd.nist.gov/vuln/detail/CVE-2020-16845)

Reply:
that's a go.mod file generated at 1.13 version timeframe.

Reply:
But we can update the version to `go 1.14` as well .